daprdocs/content/en/contributing/presentations.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "Giving a presentation on Dapr"
+linkTitle: "Presentations"
+weight: 1500
+description: How to give a presentation on Dapr and examples
+---
+
+We welcome community members giving presentations on Dapr and spreading the word about all the awesome Dapr features! We offer a template PowerPoint file to get started.
+
+{{< button text="Download the Dapr PowerPoint" link="/presentations/Dapr.zip" >}}
+
+## Giving a Dapr presentation
+
+- Begin by downloading the [Dapr Presentation Deck](/presentations/Dapr.zip). This contains slides and diagrams needed to give a Dapr presentation.
+- Next, review the docs to make sure you understand the [concepts]({{< ref concepts >}}).
+- Use the Dapr [quickstarts](https://github.com/dapr/quickstarts) repo and [samples](https://github.com/dapr/samples) repo to show demos of how to use Dapr
```

Comment:
Need a period at the end of this sentence

Reply:
Updated. Also quick tip: you can suggest a change with ctrl + g